tests/common/snappi/common_helpers.py

```diff
@@ -436,68 +449,107 @@ def config_wred(host_ans, kmin, kmax, pmax, profile=None):
             return False
 
         """ Ensure that Kmin is no larger than Kmax during the update """
-        if kmin > kmin_old:
-            host_ans.shell('sudo ecnconfig -p {} -gmax {}'.format(p, kmax))
-            host_ans.shell('sudo ecnconfig -p {} -gmin {}'.format(p, kmin))
 
+        gmax_cmd = 'sudo ecnconfig -p {} -gmax {}'
+        gmin_cmd = 'sudo ecnconfig -p {} -gmin {}'
+
+        if asic_value != 'None':
+            gmax_cmd = 'sudo ip netns exec %s ecnconfig -p {} -gmax {}' % asic_value
+            gmin_cmd = 'sudo ip netns exec %s ecnconfig -p {} -gmin {}' % asic_value
+            if asic_type == 'broadcom':
+                disable_packet_aging(host_ans, asic_value)
+
+        if kmin > kmin_old:
+            host_ans.shell(gmax_cmd.format(p, kmax))
+            host_ans.shell(gmin_cmd.format(p, kmin))
         else:
-            host_ans.shell('sudo ecnconfig -p {} -gmin {}'.format(p, kmin))
-            host_ans.shell('sudo ecnconfig -p {} -gmax {}'.format(p, kmax))
+            host_ans.shell(gmin_cmd.format(p, kmin))
+            host_ans.shell(gmax_cmd.format(p, kmax))
 
     return True
 
 
-def enable_ecn(host_ans, prio):
+def enable_ecn(host_ans, prio, asic_value='None'):
     """
     Enable ECN marking on a priority
+
     Args:
         host_ans: Ansible host instance of the device
         prio (int): priority
+        asic_value: asic value of the host
+
     Returns:
         N/A
     """
-    host_ans.shell('sudo ecnconfig -q {} on'.format(prio))
+    if asic_value == 'None':
+        host_ans.shell('sudo ecnconfig -q {} on'.format(prio))
+    else:
+        host_ans.shell('sudo ip netns exec {} ecnconfig -q {} on'.format(asic_value, prio))
 
 
-def disable_ecn(host_ans, prio):
+def disable_ecn(host_ans, prio, asic_value='None'):
```

Comment:
asic_value=None

Reply:
its actually 'None' because get_multidut_snappi_ports appends asic_value to 'None'